### PR TITLE
About env variablesAdd ENV variables to all /about content

### LINKED ISSUE #399: Add ENV variables to all /about content
- [x]  AboutContributors, e.g. [https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/AboutContributors.vue#L26](https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/AboutContributors.vue#L26)
- [x]  AboutHowItWorks
    - Phase durations, e.g. [https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/AboutHowItWorks.vue#L54](https://github.com/ethereum/clrfund/blob/develop/vue-app/src/views/AboutHowItWorks.vue#L54)

### PR DESCRIPTION
**Pull request by corwintines**
Fixes: https://github.com/ethereum/clrfund/issues/399

**What Changed**
- Using env var for nativeTokenSymbol in AboutHowItWorks

### CODE REVIEW

File: vue-app/src/views/AboutHowItWorks.vue [hunk 1]
```diff
@@ -77,7 +77,7 @@
         <links to="/verify">setup</links> before you can contribute.
       </li>
       <!-- TODO pull in variables here: max amount & native token -->
```
Comment (samajammin) [inline vue-app/src/views/AboutHowItWorks.vue]: ✂️  this comment

File: vue-app/src/views/AboutHowItWorks.vue [hunk 2]
```diff
@@ -148,7 +148,12 @@ import Links from '@/components/Links.vue'
 
 // TODO connect with state to add durations to content
```
Comment (samajammin) [inline vue-app/src/views/AboutHowItWorks.vue]: ✂️

Comment (samajammin) [reply vue-app/src/views/AboutHowItWorks.vue]: In addition - we should pull in those time variables. See the "TODO" comments in this file. e.g. "This phase will last xxx days" & "There will be a maximum of 125 projects in the round" - those amounts should both be variables we pull in dynamically.

**Review by samajammin — CHANGES_REQUESTED** [review]
These changes look good but you missed a bunch of variables on the How it Works page - see my comments.

File: vue-app/src/views/AboutHowItWorks.vue [hunk 3]
```diff
@@ -53,8 +53,9 @@
     <ul>
       <!-- TODO pull in variable here -->
       <li>This phase will last xxx days.</li>
```
Comment (corwintines) [inline vue-app/src/views/AboutHowItWorks.vue]: @samajammin trying to figure out what to do here. Dont have a good way of figuring out how long the join phase will be, just how long would be left based on the start time. Any suggestions here?

Comment (samajammin) [reply vue-app/src/views/AboutHowItWorks.vue]: Good point, we should just remove it. We won't know how long the join phase is.

Comment (corwintines) [reply vue-app/src/views/AboutHowItWorks.vue]: done!

**Review by samajammin — APPROVED** [review]
👍

**Pull request merged**